### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by the-faizmohammad**
This pull request adds responsive layout support to the existing CSS file using Flexbox & grid. The changes aim to make the desktop version of the webpage look consistent with the selected template, targeting screen widths equal to or greater than 768px.

Key Changes:

Utilized media queries to adjust the layout for different screen sizes.
Implemented Grid and Flexbox for laying out the elements on the page.
Updated styles to achieve the desired responsive behavior.
Benefits:

Enhances the user experience by providing a responsive design that adapts to different screen sizes.
Improves the visual presentation of the webpage on desktop devices, matching the selected template.
Uses modern CSS techniques (Grid and Flexbox) for efficient and flexible layouts.
Testing:

Tested the changes across different screen sizes and verified the responsive behavior.
Ensured that the elements align properly and the layout is consistent on desktop devices.
Please review and merge this pull request to incorporate the responsive layout changes, improving the desktop version of the webpage and providing a better user experience.
Please note*** facing webhint issue.

### CODE REVIEW

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: [OPTIONAL]
- [ ] Please add the button interactions as shown in the figma template requiremnt shown below. It is important to add this as it was a previous requirement for the previous milestones and it helps with awesome user experience. you can make use of the css pseudo classes such as `:active, :hover` tto implement these dsigns :+1: 
- Button Interaction
btn-i

Comment (Efiamotu-1) [inline index.html:1]: - [ ] Your design for the about me and technologies section of your portfolio website does not match the figma template as shown below.
- [ ] Your JavaScript icon is not correctly showing please kindly fix this
- [ ] There is too much spacing between the about me texts and the technologies
- [ ] There is excess spacing between the about me section and the contact me section please kindly fix this
- [ ] The Get resume button isn't displaying correctly
- > please ensure to fix all of these changes before resubmitting for a re-review :+1: :sparkles: 
- figma design
jsi
- your design
jis

Comment (Efiamotu-1) [inline index.html:1]: - [ ] The get in touch button for your contact me section is not properly centered. please kindly fix this desgin flaw :+1: 
- your design
cen

Comment (Efiamotu-1) [inline index.html:1]: - [ ] Currently your mobile design as shown below is not correctly displaying like the figma design please kindly work on correcting your design. It is important that your design matches correctly the provided figma template as this shows your attention to details as a frontend developer and this is a good skill to have :+1:
- figma design
mm
- your design
mobile

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @the-faizmohammad , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -1,270 +1,306 @@
...
+            <i class="fa-solid fa-angle-right"></i>
+          </div>
+        </li>
+        <hr />
+     
+        </li>
+      </ul>
+          
       </div>
     </section>
```
Comment (wayungi) [inline index.html:262]: - [ ] Please note that this section is not divided into halves. The **About myself** section is 40% and the **Languages** section is 60%. In your design, the each occupy half the page. Please take a look at the Figma to confirm it.

**Figma**
figma

**Your design**
about

File: index.html [hunk 2]
```diff
@@ -1,270 +1,306 @@
...
           required
           maxlength="500"
           name="message"
           cols="30"
           rows="9"
-          placeholder="Write your message here."
+          placeholder="Write your message here"
         ></textarea>
-        <button class="contact-button" type="submit">
+         <button class="my-contact-me-form-button" type="submit">
```
Comment (wayungi) [inline index.html:299]: - [ ] As suggested by the previous reviewer, the **Get In Touch** button should be centrally aligned as in the Figma.
btn

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @the-faizmohammad , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ]  Correct GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Good work adding the desktop version, the mobile version is not fully responsive at 375px. Please make sure the page fits perfectly without the horizontal scroll bar at the bottom.Horizontally scrollable sites do not offer a good user experience
mobile


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — APPROVED** [review]
## Hi @the-faizmohammad, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**